src/test/java/blackjack/domain/card/CardTest.java

```diff
@@ -0,0 +1,21 @@
+package blackjack.domain.card;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class CardTest {
+    @Test
+    @DisplayName("52장의 카드가 매번 다른 것이 나온다.")
+    void cardNotHaveDuplicate() {
+        List<Card> cards = new ArrayList<>();
+        for(int i = 1; i <= 52; i++){
+            cards.add(Card.of());
+        }
+        Assertions.assertThat(cards).doesNotHaveDuplicates();
```

Comment:
```suggestion
        for (int i = 1; i <= 52; i++) {
            cards.add(Card.of());
        }
        Assertions.assertThat(cards).doesNotHaveDuplicates();
```

`doesNotHaveDuplicates()` 👍